app/controllers/admin/events_controller.rb

```diff
@@ -9,5 +9,45 @@ def index
       # TODO: implement this method
       render status: 200, json: {}
     end
+
+    def edit
+      redirect_to new_user_session_path unless current_user
```

Comment:
You can accomplish the same thing here using Pundit's DSL: `authorize Event` (check out line 7 in this file and PR #52)

What you wrote here is totally functional, I just want to make sure we're consistent in how we do things!